platform/tests/Platform.ApiTests/packages.lock.json

```diff
@@ -565,8 +565,8 @@
       },
       "Microsoft.Bcl.AsyncInterfaces": {
         "type": "Transitive",
-        "resolved": "6.0.0",
-        "contentHash": "UcSjPsst+DfAdJGVDsu346FX0ci0ah+lw3WRtn18NUwEqRt70HaOQ7lI72vy3+1LxtqI3T5GWwV39rQSrCzAeg=="
+        "resolved": "5.0.0",
```

Comment:
chore: These seem to have been downgraded everywhere. I'm not sure which dependency change has caused this to occur, but it might be worth checking.

Reply:
Good spot. I'll check that.

Reply:
It's an implicit reference, which has result in the downgrade in a few places having sorted the explicit references.